src/views/Scan/ViewHandPage.vue

```diff
@@ -15,10 +15,64 @@
         <ion-title v-if="scanDirection">
           Scan {{ scanDirection.location }}
         </ion-title>
+        <ion-button slot="end" :disabled="isLoadingDetection" @click="setOpen(true)" style="margin-right: 10px;">
+          <ion-icon :icon="camera"></ion-icon>
+          <ion-icon  :icon="helpOutline"></ion-icon>
```

Comment:
(nitpick) formatting
```suggestion
          <ion-icon :icon="camera"></ion-icon>
          <ion-icon :icon="helpOutline"></ion-icon>
```